### ISSUE TITLE
verify_counter_repl failing

### ISSUE THREAD

**Issue by andrewjstone** [basho/riak_dt#87]
```
23:15:13.784 [info] Deployed nodes: ['dev1@127.0.0.1','dev2@127.0.0.1','dev3@127.0.0.1','dev4@127.0.0.1','dev5@127.0.0.1','dev6@127.0.0.1']
23:15:13.784 [info] Waiting for services [riak_kv] to start on ['dev1@127.0.0.1','dev2@127.0.0.1','dev3@127.0.0.1','dev4@127.0.0.1','dev5@127.0.0.1','dev6@127.0.0.1'].
23:15:23.800 [info] Wait until nodes are ready : ['dev1@127.0.0.1','dev2@127.0.0.1','dev3@127.0.0.1']
23:15:23.823 [info] [join] 'dev2@127.0.0.1' to ('dev1@127.0.0.1'): ok
23:15:23.841 [info] [join] 'dev3@127.0.0.1' to ('dev1@127.0.0.1'): ok
23:15:23.842 [info] Wait until no pending changes on ['dev1@127.0.0.1','dev2@127.0.0.1','dev3@127.0.0.1']
23:15:23.847 [info] Setting bucket properties [{allow_mult,true}] for bucket <<"test-counters">> on node 'dev1@127.0.0.1'
23:15:23.985 [info] Wait until ring converged on ['dev1@127.0.0.1','dev2@127.0.0.1','dev3@127.0.0.1']
23:15:30.254 [info] Naming cluster "A"
23:15:30.305 [info] Wait until nodes are ready : ['dev4@127.0.0.1','dev5@127.0.0.1','dev6@127.0.0.1']
23:15:30.334 [info] [join] 'dev5@127.0.0.1' to ('dev4@127.0.0.1'): ok
23:15:30.372 [info] [join] 'dev6@127.0.0.1' to ('dev4@127.0.0.1'): ok
23:15:30.372 [info] Wait until no pending changes on ['dev4@127.0.0.1','dev5@127.0.0.1','dev6@127.0.0.1']
23:15:30.374 [info] Setting bucket properties [{allow_mult,true}] for bucket <<"test-counters">> on node 'dev4@127.0.0.1'
23:15:30.414 [info] Wait until ring converged on ['dev4@127.0.0.1','dev5@127.0.0.1','dev6@127.0.0.1']
23:15:32.418 [info] Naming cluster "B"
23:15:33.448 [error] Error in process <0.105.0> on node 'riak_test@127.0.0.1' with exit value: {{badmatch,{error,{ok,"404",[{"Server","MochiWeb/1.1 WebMachine/1.10.5 (jokes are better explained)"},{"Date","Wed, 26 Mar 2014 03:15:33 GMT"},{"Content-Type","text/plain"},{"Content-Length","10"}],<<10 bytes>>}}},[{verify_counter_repl,get_counter...


23:15:33.448 [warning] verify_counter_repl failed: {{badmatch,{error,{ok,"404",[{"Server","MochiWeb/1.1 WebMachine/1.10.5 (jokes are better explained)"},{"Date","Wed, 26 Mar 2014 03:15:33 GMT"},{"Content-Type","text/plain"},{"Content-Length","10"}],<<"not found\n">>}}},[{verify_counter_repl,get_counter,1,[{file,"tests/verify_counter_repl.erl"},{line,87}]},{verify_counter_repl,confirm,0,[{file,"tests/verify_counter_repl.erl"},{line,45}]}]}
23:15:33.449 [error]
================ verify_counter_repl failure stack trace =====================
{{badmatch,{error,{ok,"404",
                      [{"Server",
                        "MochiWeb/1.1 WebMachine/1.10.5 (jokes are better explained)"},
                       {"Date","Wed, 26 Mar 2014 03:15:33 GMT"},
                       {"Content-Type","text/plain"},
                       {"Content-Length","10"}],
                      <<"not found\n">>}}},
 [{verify_counter_repl,get_counter,1,
                       [{file,"tests/verify_counter_repl.erl"},{line,87}]},
  {verify_counter_repl,confirm,0,
                       [{file,"tests/verify_counter_repl.erl"},{line,45}]}]}
==============================================================================

23:15:33.449 [notice] verify_counter_repl Test Run Complete

Test Results:
verify_counter_repl-bitcask: fail
---------------------------------------------
1 Tests Failed
0 Tests Passed
That's 0.0% for those keeping score
```

**Comment by russelldb** [comment]
See https://github.com/basho/riak_test/pull/556

**Comment by seancribbs** [comment]
This seems addressed by basho/riak_test#556. Closing.
